pkg/client/authinfo.go

```diff
@@ -0,0 +1,17 @@
+package client
+
+type AuthInfo struct {
+	code       int
+	name       string
+	token      string
+	email      string
+	body       []byte
+	permission string
+}
+
+func (a AuthInfo) Code() int          { return a.code }
```

Comment:
Do we need the `getters`? Any advantages over directly accessing the struct fields?